### PR TITLE
Fix: #135 -- Run travis tests

### PR DESCRIPTION
**Pull request by crazcalm**
Add func setDriverPaths to check if the driver paths have been set, and
if they have not been set, set them

### CODE REVIEW

File: remote_test.go [hunk 1]
```diff
@@ -44,12 +44,55 @@ var (
 
 func TestMain(m *testing.M) {
 	flag.Parse()
+	err := setDriverPaths()
+	if err != nil {
```
Comment (minusnine) [inline remote_test.go]: Combine with the above line:

```go
if err := setDriverPaths(); err != nil {
```

File: remote_test.go [hunk 2]
```diff
@@ -44,12 +44,55 @@ var (
...
+	}
+
 	s := httptest.NewServer(http.HandlerFunc(handler))
 	serverURL = s.URL
 	defer s.Close()
 	os.Exit(m.Run())
 }
 
+func setDriverPaths() error {
+	if strings.EqualFold(*selenium3Path, "") {
```
Comment (minusnine) [inline remote_test.go]: Why use EqualFold and not just ==?

File: remote_test.go (hunk 1 above)
Comment (crazcalm) [reply remote_test.go]: okay

File: remote_test.go (hunk 2 above)
Comment (crazcalm) [reply remote_test.go]: It is a bad habit of mine. Anytime I compare strings, I use strings.EqualFold. However, in this case, there is no reason not to use ==. Will do

**Comment by crazcalm** [comment]
@minusnine Just a heads up. I have already made the requested changes.

**Comment by minusnine** [comment]
Thanks! Sorry for the delay.

**Comment by crazcalm** [comment]
No worries

**Pull request merged**